### PR TITLE
[docs] Add short description of 2019 monitorama talk

### PR DESCRIPTION
**Pull request by asafm**
**What this PR does / why we need it**:
Save you the click on the lecture and viewing it, so you'll know what's it about and can focus on the videos you want to spend the time and watch

### CODE REVIEW

File: docs/overview/media.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 - [OSCON 2019: Large-Scale Automated Storage on Kubernetes](https://youtu.be/N9A7xSE9n-c) By Matt Schallert - Jul 18, 2019. [Slides](https://schallert.io/OSCON%20Large-Scale%20Automated%20Storage%20on%20Kubernetes.pdf)
 
 - [How to get the 30,000 ft view, 1 ft view and everything in between without breaking the bank](https://vimeo.com/341146220) By Martin Mao - June 5, 2019. [Slides](https://www.slideshare.net/MartinMao/monitorama-2019-pdx-martin-mao)
+  - Explains M3 Aggregator: Motivation (problems it aims to solve), the different implementations M3 went through to get to current one, and a brief high level architecture, in 30min.
```
Comment (ChrisChinchilla) [inline docs/overview/media.md:14]:
```suggestion
  - Explains M3 Aggregator: Motivation (problems it aims to solve), the different implementations M3 went through to get to the current implementation, and a brief high level architecture, in 30min.
```

**Comment by asafm** [comment]
@ChrisChinchilla Can this be merged?

**Comment by ChrisChinchilla** [comment]
@asafm yes! I wasn't aware anyone was waiting for anything further, sorry!

**Review by ChrisChinchilla — APPROVED** [review]

**Comment by ChrisChinchilla** [comment]
I'll need to figure out what is breaking these tests…

**Pull request merged**